operating-system/ubuntu/Ubuntu18.yaml

```diff
@@ -59,6 +64,15 @@ actions:
     chroot: true
     script: debos_enable_ssh.sh
 
+# Install newest kernel
+  - action: apt
+    description: Install some more packages
+    packages:
+      - linux-image-5.4.0-51-generic 
+      - linux-modules-5.4.0-51-generic
+      - linux-firmware
```

Comment:
Is this independent from updating bionic or focal?
And independent to what "latest" will be sometime?

Reply:
This is only for bionic. Kernel versions need to be revised now and then to get the latest kernel version.